package.json

```diff
@@ -46,11 +46,14 @@
     "@graphql-codegen/typescript-document-nodes": "^2.3.8",
     "@graphql-codegen/typescript-operations": "^2.5.8",
     "@graphql-codegen/typescript-react-apollo": "^3.3.7",
+    "@nestjs/schematics": "^9.0.0",
+    "@nestjs/testing": "^9.0.0",
     "@nrwl/cli": "15.3.0",
     "@nrwl/eslint-plugin-nx": "15.3.0",
-    "@nrwl/jest": "15.3.3",
+    "@nrwl/jest": "15.4.5",
```

Comment:
Can we keep the nrwl libraries on 15.3.3? We can use `nx migrate latest` to update them all and they even upgrade our code like this